channel/build.gradle

```diff
@@ -3,6 +3,6 @@ ext.moduleName = 'com.synopsys.integration.alert.channel'
 dependencies {
     implementation project(':alert-common')
     implementation project(':azure-boards-common')
-    implementation 'com.synopsys.integration:int-jira-common:1.1.0'
+    implementation 'com.synopsys.integration:int-jira-common:2.0.1-SNAPSHOT'
```

Comment:
In the process of releasing int-jira-common with the ProxyInfo change to the Builders. This will be updated as soon as the version is released.